### DISCUSSION TITLE
Generate authentication options: How can I get the user?

### DISCUSSION THREAD

**Discussion by samirspatel** [MasterKale/SimpleWebAuthn#484] [Troubleshooting]
For the endpoints `/generate-authentication-options` and `/verify-authentication`, how can I get the user when they have not logged in yet? I thought the whole point was to allow the passkey to automatically sign the user in once they have registered. 

https://github.com/MasterKale/SimpleWebAuthn/blob/master/example/index.ts#L220

@MasterKale Can you please elaborate how this `loggedInUserId` is to be retrieved when they have not yet logged in?

**Comment by MasterKale** [comment]
I'm not sure if you found previous conversations around this topic, but just in case it's still unclear I think this section of the docs might help explain things:

https://simplewebauthn.dev/docs/advanced/passkeys#remembering-challenges

If it's still not clear then please let me know how so I can unblock you and clarify the docs 😀
